chainer/functions/connection/deconvolution_nd.py

```diff
@@ -326,54 +326,99 @@ def deconvolution_nd(x, W, b=None, stride=1, pad=0, outsize=None,
+        pad (:class:`int` or :class:`tuple` of :class:`int` s):
+            Spatial padding width for input arrays
             :math:`(p_1, p_2, ..., p_N)`. ``pad=p`` is equivalent to
             ``(p, p, ..., p)``.
-        outsize (tuple of ints): Expected output size of deconvolutional
-            operation. It should be a tuple of ints
-            :math:`(out_1, out_2, ..., out_N)`. Default value is ``None`` and
-            the outsize is estimated by input size, stride and pad.
+        outsize (:class:`tuple` of :class:`int` s):
+            Expected output size of deconvolutional operation. It should be a
+            tuple of ints :math:`(l_1, l_2, ..., l_N)`. Default value is
+            ``None`` and the outsize is estimated by input size, stride and
+            pad.
         use_cudnn (bool): If ``True``, then this function uses cuDNN if
             available. Note that cuDNN supports more than one-dimensional
             deconvolution operations only.
 
     Returns:
-        ~chainer.Variable: Output variable.
-
-    The filter weight has the following dimensions
-    :math:`(c_I, c_O, k_1, k_2, ..., k_N)` which indicate the number of input
-    channels, that of output channels and the filter's spatial sizes,
-    respectively.
-
-    The one-dimensional bias vector is of size :math:`c_O`.
+        ~chainer.Variable:
+            Output variable of shape :math:`(n, c_O, l_1, l_2, ..., l_N)`.
 
-    Let :math:`X` be the input tensor of dimensions
-    :math:`(n, c_I, d_1, d_2, ..., d_N)`, :math:`(s_1, s_2, ..., s_N)` the
-    stride of filter applications, and :math:`(p_1, p_2, ..., p_N)` the spacial
-    padding size. Then the output size :math:`(out_1, out_2, ..., out_N)` is
-    determined by the following equations:
-
-    .. math::
+    .. seealso:: :class:`links.DeconvolutionND`, :func:`deconvolution_2d`
 
-        out_1 &= s_1 (d_1 - 1) + k_1 - 2 p_1,\\\\
-        out_2 &= s_2 (d_2 - 1) + k_2 - 2 p_2,\\\\
-        ...,\\\\
-        out_N &= s_N (d_N - 1) + k_N - 2 p_N.
+    .. admonition:: Example
+
+        >>> n = 10
+        >>> c_i, c_o = 3, 1
+        >>> d1, d2, d3 = 5, 10, 15
+        >>> k1, k2, k3 = 10, 10, 10
+        >>> p1, p2, p3 = 5, 5, 5
+        >>> x = np.random.uniform(0, 1, (n, c_i, d1, d2, d3)).astype('f')
+        >>> x.shape
+        (10, 3, 5, 10, 15)
+        >>> W = np.random.uniform(0, 1, (c_i, c_o, k1, k2, k3)).astype('f')
+        >>> W.shape
+        (3, 1, 10, 10, 10)
+        >>> b = np.random.uniform(0, 1, (c_o)).astype('f')
+        >>> b.shape
+        (1,)
+        >>> s1, s2, s3 = 2, 4, 6
+        >>> y = F.deconvolution_nd(x, W, b, stride=(s1, s2, s3), \
+pad=(p1, p2, p3))
+        >>> y.shape
+        (10, 1, 8, 36, 84)
+        >>> l1 = int(s1 * (d1 - 1) + k1 - 2 * p1)
+        >>> l2 = int(s2 * (d2 - 1) + k2 - 2 * p2)
+        >>> l3 = int(s3 * (d3 - 1) + k3 - 2 * p3)
+        >>> y.shape == (n, c_o, l1, l2, l3)
+        True
```

Comment:
Could you add the example that specifies `outsize` explicitly. I think it is desirable that `outsize` is different from the shape of `y` in the above example.